volatility3/cli/volshell/linux.py

```diff
@@ -40,6 +51,52 @@ def change_task(self, pid=None):
                 return None
         print(f"No task with task ID {pid} found")
 
+    def get_process(self, pid=None, offset=None):
+        """Get Task based on a process ID. Does not retrieve the layer, to change layer use the .pid attribute. The offset argument can be used both for physical or virtual offsets"""
```

Comment:
Also could you include an `Args:` and `Returns:` section here to specify the inputs and outputs please?